### ISSUE TITLE
[zwave] Unable to remove failed node

### ISSUE THREAD

**Issue by ccutrer** [openhab/openhab-addons#9537]
## Expected Behavior
Clicking "Remove Device From Controller" on the Thing's setting page should work. Then I should be able to delete the Thing, and not have it show up again.

## Current Behavior
When I click it and continue through the confirmation, the log says `NODE 80: Remove failed node failed as Controller Busy!` three times in a row. If I then delete the Thing, and go to inbox and re-scan z-wave, the Thing shows back up, because it was never removed from the actual network.

## Your Environment
* Version used: OpenHAB 3.0.0, Z-Wave binding 3.0.0
* Java 11
* Ubuntu 20.04 focal, amd64

**Comment by andrewfg** [comment]
I am curious: did you try 'Set Device as FAILed' first, before you tried 'Remove Device From Controller'

**Comment by ccutrer** [comment]
I’ve tried both ways

**Comment by andrewfg** [comment]
Reason why I asked, is that I had the same problem.  And the only way to solve it was with an external tool. 

https://community.openhab.org/t/zwave-system-migrated-on-off-sensors-now-very-slow/115493/37?u=andrewfg

**Comment by ccutrer** [comment]
Yeah, I plug my Z-stick into my windows computer and use z-wave tools there to do it. but it's a pain, and means my zwave network is completely down while I do it.

**Comment by andrewfg** [comment]
Yes, my feeling is that there is no reason why [this process description](https://community.openhab.org/t/remove-a-ghost-z-wave-node-from-habmin/6640/9) could not also be done by the OH binding (select device / send NOP / mark as failed / remove). Or??

**Comment by andrewfg** [comment]
@ccutrer it occurs to me that you may have opened your issue in the wrong place; there is a dedicated [z-wave binding repository](https://github.com/openhab/org.openhab.binding.zwave) so perhaps you should close this issue here and open it over there, where it is likely to get more attention (I suppose).

**Comment by mhilbush** [comment]
FYI there was a very long discussion on this some time ago. I don't remember whether it was on the forum, or as part of another issue. I'll try to find it.

**Comment by mhilbush** [comment]
And, yes, this should be moved to the zwave repo issue list.

**Comment by mhilbush** [comment]
Found it. FTR, here's the start of the discussion.

https://community.openhab.org/t/remove-a-ghost-z-wave-node-from-habmin/6640/94

**Comment by andrewfg** [comment]
^
@mhilbush many thanks. I have read the whole thread and it seems that the issue is the one [in this specific post](https://community.openhab.org/t/remove-a-ghost-z-wave-node-from-habmin/6640/146?u=andrewfg)

@ccutrer could you try to move your issue (this one) from openhab/addons to openhab/org.openhab.bindings/zwave ??

**Comment by mhilbush** [comment]
Yes, @andrewfg,I think you're right.

Maybe @kaikreuzer can move this issue to the zwave repo?
